test/tests_for_sdkmr/ee2_SDKMethodRunner_test.py

```diff
@@ -283,22 +276,18 @@ def test_cancel_job2(self, rq_mock, condor_mock):
         runner.get_condor = MagicMock(return_value=condor_mock)
         fixed_rj = EE2RunJob(runner)
         fixed_rj._get_module_git_commit = MagicMock(return_value="hash_goes_here")
-        fixed_rj.sdkmr.catalog_utils.list_client_group_configs = MagicMock(
-            return_value="cg goes her"
-        )
```

Comment:
This is going to be a common pattern in the tests - mocks of methods that no longer exist (since `CatalogUtils` and `Condor.extract_requirements` [or whatever is was called] are gone) are deleted but the test still passes. This is because if you look at these lines:
https://github.com/kbase/execution_engine2/pull/350/files#diff-d7d713ca6587e357238406d58fce2bf6e82ddb5339eb51dfafeaf6699103e3bfR258-R262

... a `requests` mocker is installed that returns an empty list for the client groups call for the catalog. Hence, the job requirements resolution code was formerly doubly mocked - the catalog call was mocked out, and the now deleted `CatalogUtils` and `Condor` methods were also mocked out. As is turns out, none of these tests actually depend on the value of the job requirements returned by the job requirements resolver, and so the tests work perfectly well with just the `requests` mock.